zingolib/src/wallet/transactions.rs

```diff
@@ -746,7 +754,7 @@ impl TransactionMetadataSet {
                 let nd = D::WalletNote::from_parts(
                     to.diversifier(),
                     note,
-                    Position::from(u64::MAX),
+                    None,
```

Comment:
How can we find other crazy value-initialization behaviors?

Reply:
Well, you just commented on one of them below so that's _a_ way to find them :wink: